### ISSUE TITLE
Add skip link

### ISSUE THREAD

**Issue by jpyuda** [cfpb/regulations-site#76]
There should be a link near the top of the document to skip past the TOC and go straight to 1005.1 -- this is a feature for screen reader users on old browsers. It's safe to hide the link offscreen the same way the TOC is.
